### PR TITLE
[31/10/2020] Added Drive Mechanism

### PR DESCRIPTION
**Pull request by Ashutosh781**
Drive Mechanism page added

### CODE REVIEW

**Comment by veds12** [comment]
Have you tried hosting this locally and see if the site renders properly?

File: docs/mechanical/drive_mechanism.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+* **Synchronous Drive -** This system uses synchronous rotation of its wheels to achieve motion & turns. It is made up of a system of motors. One set of which drive the wheels and the other set turns the wheels in a synchronous fashion The two sets can be directly mechanically coupled as they always move in the same direction with same speed.
+
+* **Articulated Drive -** Similar to Ackerman Steering concept, Articulated method drives a robot by deforming the entire chassis or frame to turn instead of just the wheels. This is generally used for industrial robots where a four wheeled robot is split into two, the front part and the rear part which is connected by a vertical hinge. A motor changes the angle of front part of chassis which turns the robot in a required direction and other motor drives it.
```
Comment (veds12) [inline docs/mechanical/drive_mechanism.md]: Could you add a section on reference links to books/videos/articles or any other resource related to this? For omni drive mechanism you could link omnibase

**Review by veds12 — CHANGES_REQUESTED** [review]

**Comment by Ashutosh781** [comment]
> Have you tried hosting this locally and see if the site renders properly?

Ya I did, Why are there any problems in rendering ?

File: docs/mechanical/drive_mechanism.md (hunk 1 above)
Comment (Ashutosh781) [reply docs/mechanical/drive_mechanism.md]: Should I link Omnibase Github repo ? And for other links I will try and add some papers and books.

**Comment by veds12** [comment]
There shouldn't be. Sometimes the pictures don't render properly not sure why

**Pull request merged**